### ISSUE TITLE
Support Multiple GPUs Keras style

### ISSUE THREAD

**Issue by karlmutch** [cognizant-ai-lab/studio-go-runner#52]
As an experimenter
I want to validate that allocation of multiple GPUs work
In order to support multi gpu models

This is a feature thought to work but needs verification

We want to be able to expose multiple GPUS to the keras multi GPU functions using 

from keras.utils.training_utils import multi_gpu_model

Code within the python client should then be able to do something such as ::

       G = ... array of GPU specs

	# we'll store a copy of the model on *every* GPU and then combine
	# the results from the gradient updates on the CPU
	with tf.device("/cpu:0"):
		# initialize the model
		model = MiniGoogLeNet.build(width=32, height=32, depth=3,
			classes=10)
	
	# make the model parallel
	model = multi_gpu_model(model, gpus=G)

The datacenter and google clusters are the best option for using this new feature.  When using Azure and AWS workloads are typically a single GPU.

This task is mainly about constructing a multiGPU test case as the feature was already present within the runner but never employed.

The test case will fabricate work from github assets that are json files.  The minio server present during testing locally within the runner will be referenced by the json payload and the job will be queued via the kubernetes deployed rmq server.  Resulting in a full experiment running.  Validation will be done by inspection of the result minio output.tar file which the python asssets will have used for dropping the GPUs etc used allowing us to know exactly how many GPUs were allocated and match those to any needed prometheus output.
